src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/debian/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -23,7 +24,7 @@
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll"/>
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll"/>
     <ArchitectureSpecificToolFile Include="$(BinDir)crossgen"/>
-
+    
```

Comment:
Crossgen's invariant will be to be passed the path to the JIT or have it next to it. I have confirmed it works when the JIT is next to it.

Once I add support for custom path, it will work for any location as well.